integrations/api_repo_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"testing"
 
 	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/models/db"
+	access_model "code.gitea.io/gitea/models/perm/access"
```

Comment:
I wonder why you did create a own **subpackage** instead of putting it directly into `perm`?

Reply:
Because of complicated dependent cycle. `perm` is dependent by `repo` sub package but `access` depend on it.

Reply:
that mostly means we have to move some stuff into services and or move func .... - but we can do that later too